src/accel_group.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use gdk;
+use glib;
+use glib::object::{Cast, IsA};
+use glib::translate::*;
+use glib::ToValue;
+use gtk_sys;
+use AccelFlags;
+use AccelGroup;
+
+pub trait AccelGroupExtManual: 'static {
+    fn connect_accel_group<F>(
+        &self,
+        accel_key: u32,
```

Comment:
I'm wondering: shouldn't there be an enum for this one somewhere?

Reply:
Not that I could find, it's using `u32` everywhere.

Reply:
Weird that there doesn't seem to be an enum anywhere... That's quite ugly. Well, whatever...